### ISSUE TITLE
GoogleImageCrawler failed

### ISSUE THREAD

**Issue by cswhjiang** [hellock/icrawler#94]
The test script:
```
from icrawler.builtin import GoogleImageCrawler

google_crawler = GoogleImageCrawler(storage={'root_dir': 'your_image_dir'})
google_crawler.crawl(keyword='cat', max_num=100)

```

The errors:
```
2021-02-22 14:45:15,286 - INFO - icrawler.crawler - start crawling...
2021-02-22 14:45:15,287 - INFO - icrawler.crawler - starting 1 feeder threads...
2021-02-22 14:45:15,288 - INFO - feeder - thread feeder-001 exit
2021-02-22 14:45:15,288 - INFO - icrawler.crawler - starting 1 parser threads...
2021-02-22 14:45:15,289 - INFO - icrawler.crawler - starting 1 downloader threads...
2021-02-22 14:45:15,742 - INFO - parser - parsing result page https://www.google.com/search?q=cat&ijn=0&start=0&tbs=&tbm=isch
Exception in thread parser-001:
Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/icrawler/parser.py", line 104, in worker_exec
    for task in self.parse(response, **kwargs):
TypeError: 'NoneType' object is not iterable

2021-02-22 14:45:20,293 - INFO - downloader - no more download task for thread downloader-001
2021-02-22 14:45:20,294 - INFO - downloader - thread downloader-001 exit
2021-02-22 14:45:20,298 - INFO - icrawler.crawler - Crawling task done!

```

**Comment by ZhiyuanChen** [comment]
Could you please update to the latest version and see if the error still exists

**Comment by Patrick-wlz** [comment]
The update in #93 totally solved this problem for me.

**Comment by cswhjiang** [comment]
> Could you please update to the latest version and see if the error still exists

Problem solved. Thanks.

**Comment by ZhiyuanChen** [comment]
fixed by #93

**Comment by babanthierry94** [comment]
fixed by #93 
It works fine
